psi4/driver/driver_nbody.py

```diff
@@ -25,830 +25,1290 @@
 #
 # @END LICENSE
 #
-
+"""Plan, run, and assemble QC tasks to obtain many-body expansion and basis-set superposition error treatments.
+
+=============
+ManyBody Flow
+=============
+Bullet points are major actions
+Lines of dashes denote function calls
+e/d/dd=dg/g/h := energy, dipole, dipole derivative = dipole gradient, gradient, Hessian
+
+---------------------------
+ManyBodyComputer.__init__()
+---------------------------
+* not an explicit function but pydantic handles some defaults and validation
+* fields molecule, nfragments, bsse_type, return_total_data, and initial max_nbody set
+* BaseComputer.__init__()
+
+task_planner()
+--------------
+* computer gets modified from task_planner outside this file!
+* user levels kwarg processed into modelchem treatment levels and the n-body levels they cover. fields nbodies_per_mc_level set and max_nbody reset
+* for each modelchem treatment level, call build_tasks() below via one of four routes, depending on simple MB or layered MB(FD), MB(CBS), or MB(FD(CBS))
+
+    ------------------------------
+    ManyBodyComputer.build_tasks()
```

Comment:
If this is build _tasks_, why does it say a supersystem _or_ otherwise? Some tasks are supersystem, some are otherwise. Have I missed something?

Reply:
I'm not quite following, but I can think of two potential points for confusion:
* (1a) there's "supersystem", the whole input molecule ignoring fragmentation
* (1b) there's "supersystem", the user kwarg that signifies treating all-the-rest n-body levels with a certain modelchem
* (2a) `build_tasks` is called in a loop, so each call is dumping more tasks into the task set

Reply:
can't really, as `build_tasks` is a general fn inherited from BaseComputer